docs/virtual-env.md

```diff
@@ -4,7 +4,7 @@
 
 - [Virtualenv](http://www.virtualenv.org/en/latest/)
 - [Python 3](http://www.python.org/) (Can be installed using `brew install python3`)
```

Comment:
3.10 is the latest supported by this version of uwsgi, I believe. And it's what's in the dockerfile, so we should put something in about using a supported version, perhaps further on.

Also, brew is perhaps not the best choice if we want people to be flexible on version and have different versions for different repos. Suggest we recommend pyenv - could crib from: https://github.com/ministryofjustice/cla_backend/blob/master/VIRTUAL_ENV.md#pyenv-python2 and could add a `.python-version` file